### PR TITLE
Create Build Src folder and base gradle file

### PR DESCRIPTION
**Pull request by Bilijin**

### CODE REVIEW

File: base-module.gradle [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    buildFeatures {
+        compose true
+        viewBinding true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = Kotlin.compilerExtensionVersion
+    }
+    packagingOptions {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
```
Comment (eunix56) [inline base-module.gradle:42]: I'm curious about what this does

File: build.gradle.kts [hunk 2]
```diff
@@ -1,11 +1,7 @@
...
-    }
-}// Top-level build file where you can add configuration options common to all sub-projects/modules.
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id("com.android.application") version "7.3.1" apply false
-    id("com.android.library") version "7.3.1" apply false
-    id("org.jetbrains.kotlin.android") version "1.6.10" apply false
+//    id("com.android.application") version "7.3.1" apply false
+//    id("com.android.library") version "7.3.1" apply false
+//    id("org.jetbrains.kotlin.android") version "1.6.10" apply false
```
Comment (eunix56) [inline build.gradle.kts:5]: Is there a reason why these were commented out? Were they supposed to be deleted?

**Comment by eunix56** [comment]
I think all in all, the changes you made were really nice 👍. Good job

File: build.gradle.kts (hunk 2 above)
Comment (Bilijin) [reply build.gradle.kts:5]: Yeah they were supposed to be deleted. I commented them out initially to make sure the project compiled fine without them

File: base-module.gradle (hunk 1 above)
Comment (Bilijin) [reply base-module.gradle:42]: It's the base gradle file that all the other build.gradle files inherit from. To reduce repeated code

**Pull request merged**